product_docs/docs/biganimal/release/using_cluster/03_modifying_your_cluster/06_pgbouncer_configuration.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Configuring EDB PgBouncer
```

Comment:
```suggestion
title: Using PgBouncer to manage your connections
```
Not sure this is exactly what PgBouncer does, but we want a title that reflects the purpose/value of using PgBouncer